src/marqo/version.py

```diff
@@ -1,4 +1,5 @@
-__version__ = "2.9.0"
+__version__ = "2.10.0"
```

Comment:
Is this used for testing? Should this be changed during the release process?

Reply:
already changed in a previous pr, so this shouldnt be an issue. it's also used for docs tests.